examples/python/ocr/README.md

```diff
@@ -0,0 +1,156 @@
+        return self.value[0]  # Returns the numerical identifier
+
+    @property
+    def type(self):
+        return self.value[1]  # Returns the type
+
+    @property
+    def color(self):
+        return self.value[2]  # Returns the color
+
+    @classmethod
+    def get_annotation(cls):
+        return [(layout.number, layout.type, layout.color) for layout in cls]
+
+
+def detect_and_log_layout(img_path):
+    rr.log(
+        "Image",
+        # The annotation is defined in the Layout class based on its properties
+        rr.AnnotationContext(LayoutType.get_annotation()),
+        timeless=True
+    )
+```
+
+### Detections
+The detections include the layout types and the text detections. Both of them are logged as [`Boxes2D`](https://www.rerun.io/docs/reference/types/archetypes/boxes2d) to Rerun.
+
+```python
+rr.log(
+    base_path,
+    rr.Boxes2D(
+        array=record['bounding_box'],
+        array_format=rr.Box2DFormat.XYXY,
+        labels=[str(layout_type.type)],
+        class_ids=[str(layout_type.number)]
+    ),
+    rr.AnyValues(name=record_name),
+    timeless=True
+)
+```
+In the detection of the text, the detection id and the confidence are given supplementary.
+```python
+rr.log(
+    f"{base_path}/Detections/{detection['id']}",
+    rr.Boxes2D(
+        array=detection['box'],
+        array_format=rr.Box2DFormat.XYXY,
+        class_ids=[str(layout_type.number)]
+    ),
+    rr.AnyValues(
+        DetectionID=detection['id'],
+        Text=detection['text'],
+        Confidence=detection['confidence']
+    ),
+    timeless=True
+)
+```
+
+### Setting up the blueprint
+
+The blueprint for this example is created by the following code:
+
+```python
+rr.send_blueprint(rrb.Blueprint(
+        rrb.Vertical(
+            rrb.Horizontal(
+                rrb.Spatial2DView(name="Layout", origin='Image/', contents=["Image/**"] + detections_paths),
+                rrb.Spatial2DView(name="Detections", contents=["Image/**"]),
+                rrb.TextDocumentView(name="Recovery", contents='Recovery')
+            ),
+            rrb.Horizontal(
+                *tabs
+            ),
+            row_shares=[4, 3],
+        ),
+        collapse_panels=True,
+    )
+)
+```
+
+## Run the code
+To run this example, make sure you have the Rerun repository checked out and the latest SDK installed:
+```bash
+$ pip install --upgrade rerun-sdk  # install the latest Rerun SDK
+$ git clone git@github.com:rerun-io/rerun.git  # Clone the repository
+$ cd rerun
+$ git checkout latest  # Check out the commit matching the latest SDK release
+```
+Install the necessary libraries specified in the requirements file:
+```bash
+$ pip install -e examples/python/ocr
+```
+To experiment with the provided example, simply execute the main Python script:
+```bash
+$ python -m ocr # run the example
+```
+If you wish to explore additional features, or save it use the CLI with the `--help` option for guidance:
+```bash
+$ python -m ocr --help
+```
```

Comment:
```suggestion
```bash
pip install --upgrade rerun-sdk  # install the latest Rerun SDK
git clone git@github.com:rerun-io/rerun.git  # Clone the repository
cd rerun
git checkout latest  # Check out the commit matching the latest SDK release
```
Install the necessary libraries specified in the requirements file:
```bash
pip install -e examples/python/ocr
```
To experiment with the provided example, simply execute the main Python script:
```bash
python -m ocr # run the example
```
If you wish to explore additional features, or save it use the CLI with the `--help` option for guidance:
```bash
python -m ocr --help
```
```